07-routing-gossip.md

```diff
@@ -300,6 +310,11 @@ The following table specifies the meaning of the individual bits:
 The creating node MUST set `signature` to the signature of the
 double-SHA256 of the entire remaining packet after `signature` using its own `node_id`.
 
+The creating node MUST set `chain_hash` to match 32-byte hash that
+uniquely identifies the channel within the `channel_announcement` message.
```

Comment:
I guess this should be the tuple (`chain_hash`, `short_channel_id`) which uniquely identifies the channel.